README.md

```diff
@@ -24,36 +28,84 @@ Supported Features
    * Setting the IPv4 address and netmask of the device;
    * Setting the persistence mode of the device;
    * Setting the name of the device (Linux only);
-   * Setting the description of the device (OpenBSD and FreeBSD only).
-
-Note for Mac OS X users
------------------------
-
-You need to install the tuntaposx project for this library to be useful,
-which is a third-party kext.
+   * Setting the description of the device (OpenBSD and FreeBSD only);
+   * Wrapper libraries for other languages.
 
-Supported Systems
------------------
+#### Supported Systems
 
    * OpenBSD;
    * Linux;
    * NetBSD;
    * Darwin (up to High Sierra).
 
-Current Porting Efforts
------------------------
+#### Current Porting Efforts
 
    * Windows;
    * FreeBSD.
 
-In the future
--------------
+#### In the future
 
    * AIX;
    * Solaris.
 
-License
--------
+## Install
+
+#### Requires
+
+* cmake;
+* C and C++ compilers.
+
+#### Build
+
+This project is built with cmake:
+
+    $ mkdir build; cd build
+    $ cmake ../
+    $ make
+
+The following options can be tweaked:
+
+- `ENABLE_CXX`: Enable building of the C++ wrapper library libtuntap++;
+- `ENABLE_PYTHON`: Enable building of the Python wrapper library pytuntap;
+- `BUILD_TESTING`: Enable building of the regress tests;
+
+#### Other languages bindings
+
+We currently provide wrappers for two other languages: C++ and Python,
+respectively named libtuntp++ and pytuntap.  More instructions about them is
+provided in the [`bindings`](bindings/README.md) folder.
+
+The C++ library is built by default and can be disabled with the flag
+`ENABLE_CXX`.
+
+The Python library is disabled by default and requires both `ENABLE_CXX` and
+`ENABLE_PYTHON` to work.  You will also need Python 3.6 and Boost libraries.
```

Comment:
Maybe we should not specify the minor release of python ? I think we are compatible with python3/2 in general, no ?

Reply:
I just documented the existing build: the file `bindings/python/CMakeLists.txt` has a line with `find_package(... python36)`.

So I don't really know the level of compatibility of pytuntap as I am pretty sure I never tested it. You only can tell me about it 😄.